src/format.c

```diff
@@ -21,18 +21,22 @@
  */
 
 #include "format.h"
-
 #ifndef LIBXMP_NO_PROWIZARD
 #include "loaders/prowizard/prowiz.h"
 #endif
 
 const struct format_loader *const format_loaders[NUM_FORMATS + 2] = {
 	&libxmp_loader_xm,
 	&libxmp_loader_mod,
+#ifndef LIBXMP_CORE_PLAYER
 	&libxmp_loader_flt,
 	&libxmp_loader_st,
+#endif
+#ifndef LIBXMP_CORE_DISABLE_IT
 	&libxmp_loader_it,
+#endif
 	&libxmp_loader_s3m,
+#ifndef LIBXMP_CORE_PLAYER
```

Comment:
IMO just reorder IT and S3M, it doesn't need to be this messy.

Reply:
I didn't want to change the order just in case, but I'll happily do it.

Reply:
And done: 91d1e57ecb80fc2cef5b87881ab082d3248c8e24